src/main/java/org/elasticsearch/app/api/server/exceptions/CouldNotSearchForIndex.java

```diff
@@ -0,0 +1,26 @@
+package org.elasticsearch.app.api.server.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.CONFLICT)
+public class CouldNotSearchForIndex extends RuntimeException {
+    public CouldNotSearchForIndex() {
+    }
+
+    public CouldNotSearchForIndex(String message) {
+        super(message);
+    }
+
+    public CouldNotSearchForIndex(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public CouldNotSearchForIndex(Throwable cause) {
+        super(cause);
+    }
+
+    public CouldNotSearchForIndex(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
```

Comment:
All of the exceptions have this method with 4 parameters but you never use it in the code, so what is the purpose of duplicating all constructors of RuntimeException?

Reply:
Noted